docs/contributing-documentation.md

```diff
@@ -1,54 +1,99 @@
 
 ### Branches and versions
 
 The `master` branch is the "work in progress" version of the HTML specification. It is available at [https://w3c.github.io/html/](https://w3c.github.io/html/)
 
-Once a year, the HTML editors create a new `<version>` branch for the HTML specification. It only contains features that the Working Group believes can be shipped as part of the W3C Recommendation. That branch becomes associated with a specific version of the HTML specification. For a *limited period* of time, the Editor Team only accepts editorial changes or removal of features at risks in this branch. It becomes frozen once that version of HTML becomes a W3C Recommendation. Unless you're targetting a specific version of HTML (and really, you shouldn't), pull requests MUST always be made against the `master` branch.
+About once a year, the HTML editors create a new `<version>` branch for the HTML specification. It only contains features that the Working Group believes can be shipped as part of the W3C Recommendation. 
+That branch becomes associated with a specific version of the HTML specification. For a *limited period* of time, the Editor Team only accepts editorial changes or removal of features at risks in this branch.
+It becomes frozen once that version of HTML becomes a W3C Recommendation. 
+Unless you're targetting a specific version of HTML (and really, you shouldn't), pull requests MUST always be made against the `master` branch.
+
+### Editorial changes
+
+Editorial changes include spelling and grammar improvements, and providing non-normative clarification,
+as well as making normative text easier to read and understand. Conformance of tools or content is not changed by an editorial change.
+
+Where the specification is inconsistent, e.g. the main text says the `<exampleElement>` element must not have a `title` attribute,
+but the table of attributes says the `title` attribute is valid for the element, making it consistent is generally an editorial change.
+
+1. Identify an [HTML issue](https://github.com/w3c/html/issues) that you want to work on. If no issue exists, please create one.
+
+2. Make a branch and commit your edits to it. 
+Please remember to add your name in the [Acknowledgements section](https://github.com/w3c/html/blob/master/sections/acknowledgements.include)
+if it is not there already.
+(We are grateful for contributions, and even more so if the editors don't have to make a separate change to do that :) ).
+
+3. For large or complicated changes, please run Bikeshed to make sure there are no errors.
+Note: The [Build documentation](docs/build-documentation.md) page has information about setting up and using Bikeshed.
+
+4. Create a Pull Request and add at least the label "Editorial". If you are not a member of the Working Group, you will get a notice that your Pull Request failed an IPR check.
```

Comment:
This isn't really very clear. Either someone should add the editorial label to an editorial commit or not. We should be clear.